### DISCUSSION TITLE
Tabs in the Data Catalog

### DISCUSSION THREAD

**Discussion by desilvarami** [TerriaJS/terriajs#6827] [Q&A] [answered]
Hi,
I wish to have number of tabs in the Data Catalog. As discussed in the following thread, I have set _"tabbedCatalog": true_, in config.json. 

#5768 

But still I don't get the desired output. Please refer the following image.
![image](https://github.com/TerriaJS/terriajs/assets/6550207/0b4ab14d-dcb3-4aeb-ac73-76341c74d0e0)

Are there any other settings that I need to change? Please help?

Kind Regards
Ramesh

**Comment by sixlighthouses** [comment, marked as answer]
Hi Ramesh,

to enable tabs in the Data Catalogue the attribute

```
"tabbedCatalog": true,
```
should be added in config.json under "parameters" : {

To test I added it below appName 

```
"appName": "Terria Map",
"tabbedCatalog": true,
```
Then within your Initialization file -- https://docs.terria.io/guide/customizing/initialization-files/

Add groups - like the example below ("My group"), the "members" then contains an array of the items you wish to display in the Data Catalogue

```{
    "catalog": [
        {
            "type": "group",
            "name": "My group",
            "members": [
            ...
            ]
        },
        ...
    ],
    "homeCamera": {
        "north": -8,
        "east": 158,
        "south": -45,
        "west": 109
    },
    "initialCamera": { ... },
    "corsDomains": [ "myserver.gov.au" ],
    "baseMaps": {
      "items": [
        {
          "item": {
            "id": "basemap-darkmatter",
            "name": "Dark Matter",
              ...
          },
          "image": "/images/dark-matter.png"
        }
      ],
    },
    ...
}
```

Screenshot showing 2 tabs generated from 2 groups I created within my Initialization file

<img width="924" alt="image" src="https://github.com/TerriaJS/terriajs/assets/1597830/8f155959-7174-4f79-81ba-b79be0d98a09">

hopefully that helps, please let us know how you get on

cheers
Brendon

**Reply by desilvarami** [reply]
Dear Brendon, 
Thank you very much for the instructions. It perfectly worked. I had not configured the attribute "tabbedCatalog" under parameters but just below the "initializationUrls". Now I am getting the desired output as shown in the following image. Really appreciate your assistance by providing the pinpoint answer.

![image](https://github.com/TerriaJS/terriajs/assets/6550207/867af9d1-0466-4794-83bf-1b3af60235c4)

**Reply by sixlighthouses** [reply]
Good to hear Ramesh
